api/src/main/java/org/pmiops/workbench/elasticsearch/ElasticFilters.java

```diff
@@ -241,7 +326,9 @@ private static boolean isStandardConcept(SearchParameter param) {
                 .map(id -> Long.toString(id))
                 .collect(Collectors.toSet()));
       } else {
-        out.add(Long.toString(param.getConceptId()));
+        if (param.getConceptId() != null) {
+          out.add(Long.toString(param.getConceptId()));
```

Comment:
nit: Could you add a comment here explaining that some search parameters are attribute/modifiers only.

Reply:
Done.